### ISSUE TITLE
Please configure debian:buster-slim so it's more compatible with Redis

### ISSUE THREAD

**Issue by ruralcoder** [docker-library/redis#224]
I get the following warnings 

```
# WARNING: The TCP backlog setting of 511 cannot be enforced because /proc/sys/net/core/somaxconn is set to the lower value of 128.
# Server initialized
# WARNING overcommit_memory is set to 0! Background save may fail under low memory condition. To fix this issue add 'vm.overcommit_memory = 1' to /etc/sysctl.conf and then reboot or run the command 'sysctl vm.overcommit_memory=1' for this to take effect.
# WARNING you have Transparent Huge Pages (THP) support enabled in your kernel. This will create latency and memory usage issues with Redis. To fix this issue run the command 'echo never > /sys/kernel/mm/transparent_hugepage/enabled' as root, and add it to your /etc/rc.local in order to retain the setting after a reboot. Redis must be restarted after THP is disabled.
```

Of course the `debian:buster-slim` base is so restricted and uses RO filesystem, it makes this dockerfile not so user-friendly.

**Comment by tianon** [comment]
Duplicate of #14, #19, #20, #55, #165 (to name a few).

TL;DR, these need to be set on the *host*, not in the container (if they were image settings, we would set them by default).

**Comment by ruralcoder** [comment]
Do you mean the container uses these settings from the host? Will try. Awesome.

**Comment by ruralcoder** [comment]
On the host machine I get
```
$ cat /proc/sys/net/core/somaxconn
65535
```

When I run docker I still see
```
# WARNING: The TCP backlog setting of 511 cannot be enforced because /proc/sys/net/core/somaxconn is set to the lower value of 128.
```

When I bash into the docker 
```
docker run -it dmvmonkey/redis  /bin/bash
```

```
# cat /proc/sys/net/core/somaxconn
128
```

I don't see it carrying from the host into the container.

Any further advice??

**Comment by yosifkit** [comment]
```console
$ docker run -it --rm debian
root@2a9ac928345d:/# cat /proc/sys/net/core/somaxconn
128
root@2a9ac928345d:/# exit
$ docker run -it --rm --sysctl net.core.somaxconn=11223344 debian
root@085d23ea47ae:/# cat /proc/sys/net/core/somaxconn
11223344
```
